octobot_trading/exchanges/implementations/spot_ccxt_exchange.py

```diff
@@ -138,7 +138,7 @@ async def _verify_order(self, created_order, order_type, symbol, price, params=N
 
         # on some exchange, market order are not including price, add it manually to ensure uniformity
         if created_order[ecc.PRICE.value] is None and price is not None:
-            created_order[ecc.PRICE.value] = float(price)
+            created_order[ecoc.PRICE.value] = float(price)
```

Comment:
why ? it should be parsed later on using ccxt values I guess ?

Reply:
In my opinion `clean_order` is not a parser so we expect that the returned value of _verify_order is an unified Order

Reply:
`clean_order` is called right after but here we are in verify_order, we are still manipulating pure ccxt values